api/lib/handlers/login/index.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var config = require('../../config.js').config;
+var pg = require('pg');
+
+module.exports.post = function(req, res, next) {
+  var id = req.body.id;
+  var pool = new pg.Pool(config);
+  pool.connect(function(err, client, done) {
+    if(err) {
+      return console.error('error fetching client from pool', err);
```

Comment:
I'm not sure that exposing this error publicly is a good idea.
We should log this and return a standard error to the client.